test/type_prop/non_zero.cpp

```diff
@@ -24,7 +24,7 @@ using namespace ngraph;
 TEST(type_prop, non_zero)
 {
     auto data = make_shared<op::Parameter>(element::f32, Shape{3, 3, 224, 224});
-    auto non_zero = make_shared<op::NonZero>(data);
+    auto non_zero = make_shared<op::v3::NonZero>(data, element::i64);
```

Comment:
I think that we need to keep existing tests and add new tests for v3 version of NonZero

Reply:
There is no version of NonZero other than the v3::NonZero I have modified. It was necessary to adjust the existing tests to the new implementation.